common/static/common/js/karma.common.conf.js

```diff
@@ -0,0 +1,216 @@
+// Common configuration for Karma
+/* jshint node: true */
+'use strict';
```

Comment:
I'd wrap all of this in a self-executing anonymous function. This is good practice in general to keep the global namespace clean.